README.adoc

```diff
@@ -9,12 +9,16 @@ image::https://badges.gitter.im/Join%20Chat.svg[link="https://gitter.im/jruby-gr
 
 You can also join us on the link:https://groups.google.com/forum/#!forum/jruby-gradle[JRuby/Gradle mailing list]
 
-The purpose of these plugins is to bundle useful link:http://gradle.org[Gradle]
-functionality for JRuby projects. Use of this plugin replaces the need for both link:http://bundler.io[Bundler] and link:https://github.com/jruby/warbler[Warbler].
+JRuby/Gradle brings the power and flexibilitye of 
+link:http://gradle.org[Gradle]
```

Comment:
You can probablyhave everything on a single line. This is pretty much Asciidoc writing convention. Most editors can wrap long lines and INtelliJ actually has a setting which does this automatically for`.doc` files.

Reply:
This is a style pattern we use in the Jenkins documentation to make life a little bit easier, putting links on their own lines